src/NServiceBus.Persistence.AzureStorage/Timeout/TimeoutLogic/TimeoutPersister.cs

```diff
@@ -151,79 +138,28 @@ public TimeoutPersister(string timeoutConnectionString, string timeoutDataTableN
             var now = DateTime.UtcNow;
 
             var timeoutDataTable = client.GetTableReference(timeoutDataTableName);
-            var timeoutManagerDataTable = client.GetTableReference(timeoutManagerDataTableName);
 
-            await TryUpdateSuccessfulRead(timeoutManagerDataTable).ConfigureAwait(false);
-
-            var lastSuccessfulReadEntity = await GetLastSuccessfulRead(timeoutManagerDataTable).ConfigureAwait(false);
-            var lastSuccessfulRead = lastSuccessfulReadEntity?.LastSuccessfullRead;
-
-            TableQuery<TimeoutDataEntity> query;
-
-            if (lastSuccessfulRead.HasValue)
-            {
-                query = new TableQuery<TimeoutDataEntity>()
-                    .Where(
+            var query = new TableQuery<TimeoutDataEntity>()
+                .Where(
+                    TableQuery.CombineFilters(
                         TableQuery.CombineFilters(
-                            TableQuery.CombineFilters(TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.GreaterThanOrEqual, lastSuccessfulRead.Value.ToString(partitionKeyScope)),
-                                TableOperators.And,
-                                TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.LessThanOrEqual, now.ToString(partitionKeyScope))),
+                            TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.GreaterThanOrEqual, now.AddYears(-1).ToString(partitionKeyScope)),
```

Comment:
I do not like this huge range one bit, it will lead to horrible performance as it has a potential 8765 partitions to scan. So a potential of 8765 queries to execute in a serial way, it wil take ages to start an endpoint that has been down for a while.